Docs/install/index.rst

```diff
@@ -34,118 +34,175 @@
+- :ref:`Installation of the latest version for all platforms from release packages <idx-install-latest>`
+- :ref:`Installation of older versions <idx-install-older>`
+
+You can also:
 
-The AIMET PyTorch GPU PyPI packages are available for environments that meet the following requirements:
+- :doc:`Install manually <install_host>`
+- :doc:`Install a Docker image, including from a pre-built image or local build <install_docker>`
+
+.. note::
+    
+    The following system requirements apply to all installations.
+
+System requirements
+===================
+
+The AIMET package requires the following host platform setup:
 
 * 64-bit Intel x86-compatible processor
-* Linux Ubuntu 22.04 LTS [Python 3.10] or Ubuntu 20.04 LTS [Python 3.8]
+* Linux Ubuntu 22.04 LTS (Python 3.10) or Ubuntu 20.04 LTS (Python 3.8)
+* bash command shell
+* For GPU variants:
+    * Nvidia GPU card (Compute capability 5.2 or later)
+    * Nvidia driver version 455 or later (using the latest driver is recommended; both CUDA and cuDNN are supported)
+    * nvidia-docker - Installation instructions: https://github.com/NVIDIA/nvidia-docker
+
+.. _idx-install-quick:
+
+Quick Install
+=============
+
+The fastest way to install AIMET is to use the AIMET PyTorch GPU PyPI packages.
+
+Prerequisites
+-------------
+
+The following software versions are required for the quick install:
+
 * CUDA 12.0
 * Torch 2.2.2
 
-**Pip install**
+Ensure that you have the LAPACK linear algebra package installed:
 
 .. code-block:: bash
 
-    apt-get install liblapacke libpython3-dev
+    apt-get install liblapacke
+
+Installation
+------------
+
+- **Type the following command to install AIMET using Pip:**
+
+.. code-block::
+
     python3 -m pip install aimet-torch
 
 
-Release Packages
-~~~~~~~~~~~~~~~~
+Next steps
+----------
 
-For other AIMET variants, install the *latest* version from the .whl files hosted at https://github.com/quic/aimet/releases (compatible with python 3.10 only at this time).
+See the :doc:`Quantization User Guide </user_guide/model_quantization>` for a discussion of how to use AIMET quantization.
 
-**Prerequisite**
+See the :doc:`Examples Documentation </user_guide/examples>` to try AIMET on example quantization and compression problems.
 
-The following pre-requisites apply to all variants. The GPU variants may need additional packages - please see `Advanced Installation Instructions`_ for details.
 
+.. _idx-install-latest:
 
-.. code-block:: bash
+Installing  the latest version with release packages
+====================================================
+
+Install the latest version of any AIMET variant from the **.whl** files hosted at https://github.com/quic/aimet/releases.
 
-    # Install pre-requisite packages
-    apt-get install liblapacke libpython3-dev
+Choose and install a package
+----------------------------
 
-    # Install an compatible version of pip (since the latest version is NOT compatible with our wheel packages)

```

Comment:
We cannot delete this versioned pip install step. Could you please review this recent commit and ensure that no changes from there got dropped as part of the current PR?
https://github.qualcomm.com/qualcomm-ai/aimet/commit/f77040bbfe13d08e49635db1861370af05b7e154

Reply:
Reviewed. Fixed torch_pkg_url and find_pkg_url_str instructions in host and docker install procedures.